homeassistant/components/todoist/calendar.py

```diff
@@ -94,7 +94,7 @@
                             ),
                             vol.Optional(
                                 CONF_PROJECT_LABEL_WHITELIST, default=[]
-                            ): vol.All(cv.ensure_list, [vol.All(cv.string, vol.Lower)]),
+                            ): vol.All(cv.ensure_list, [vol.All(cv.string)]),
```

Comment:
Labels are case sensitive, so it's fine to have `Test` or `test` but they are treated as different labels in todoist.  Because of this we need to remove the lower-casing of any provided labels.